README.md

```diff
@@ -47,7 +47,8 @@ Check out our [installation instructions](http://www.git-town.com/install.html)
 
 Each command can be [aliased](https://git-scm.com/book/en/v2/Git-Basics-Git-Aliases) individually to remove the `town` prefix with:
 ```
-git config --global alias.hack town-hack
+git config --global alias.hack 'town hack' # For Git-Town versions >= 4
+git config --global alias.hack town-hack # For Git-Town version < 4
```

Comment:
I'd prefer we just have documentation for the latest on the README. It was mentioned in the RELEASE_NOTES.md that the aliases changed